emulator/pragma.go

```diff
@@ -0,0 +1,99 @@
+/*
+ * Flow Emulator
+ *
+ * Copyright 2019 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package emulator
+
+import (
+	"regexp"
+)
+
+var (
+	PragmaDebug      = "debug"
+	PragmaSourceFile = "sourceFile"
+)
+
+type PragmaList []Pragma
+
+var pragmaRegexp = regexp.MustCompile(`#([a-zA-Z_]([0-9a-zA-Z_]*)?)(\(\"([^\"]*?)\"\))?`)
```

Comment:
can you add a comment with a few examples? Regexes are hard to read.

Reply:
added in https://github.com/onflow/flow-emulator/pull/437/commits/e6360c624328bcfc657ec1f4fbce5acd8e2fbb86